<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZrmWidget</class>
 <widget class="QWidget" name="ZrmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>521</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBoxMainDisplay">
       <property name="title">
        <string>Параметры</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="ZrmMainDisplay" name="zrm_display" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBoxMethod">
       <property name="title">
        <string>Метод</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="ZrmMethodBase" name="zrm_method" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBoxLogerChart">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="ZrmLogerChartUI" name="tabChart">
           <attribute name="title">
            <string>График</string>
           </attribute>
          </widget>
          <widget class="ZrmCellView" name="tabCell">
           <attribute name="title">
            <string>АКБ</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBoxDevice">
       <property name="title">
        <string>Подключенные устройства</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="ZrmReadySlaveWidget" name="zrm_ready" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZrmCellView</class>
   <extends>QWidget</extends>
   <header>zrmcellview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZrmMethodBase</class>
   <extends>QWidget</extends>
   <header>zrmmethodbase.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZrmMainDisplay</class>
   <extends>QWidget</extends>
   <header>zrmmaindisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZrmReadySlaveWidget</class>
   <extends>QWidget</extends>
   <header>ZrmReadySlaveWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZrmLogerChartUI</class>
   <extends>QWidget</extends>
   <header>ZrmLogerChartUI.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
